packages/cra-template-mephisto-review/template/src/components/css/WordCloud.css

```diff
@@ -1,9 +1,9 @@
-.cloud {
+.word-cloud {
   text-align: center;
   justify-content: center;
 }
 
-.cloud p {
+.word-cloud p {
```

Comment:
This css file should prob also go in the plugins folder?

Reply:
Same as the above comment, this css file is for the unit word cloud component